pkg/scheduler/metrics/metrics.go

```diff
@@ -65,7 +65,14 @@ var (
 			Help:           "Number of attempts to schedule pods, by the result. 'unschedulable' means a pod could not be scheduled, while 'error' means an internal scheduler problem.",
 			StabilityLevel: metrics.STABLE,
 		}, []string{"result", "profile"})
-
+	ScheduleAttemptsToSchedule = metrics.NewHistogramVec(
+		&metrics.HistogramOpts{
+			Subsystem:      SchedulerSubsystem,
+			Name:           "schedule_attempts_to_schedule_total",
```

Comment:
`_total` is used for Counters.

This is a distribution. Maybe just `schedule_attempts_to_schedule` is fine.